tests/test-github-policy.sh

```
@@ -609,6 +609,18 @@ else
   ok "bootstrap deleted nothing"
 fi
 
+echo "==> A queue-less policy still enables auto-merge (touchstone pr merge arms it)"
+rm -f "$TMP_DIR/state/branch.json" "$TMP_DIR/state/ruleset.json" "$TMP_DIR/state/repo-ruleset.json" "$TMP_DIR/state/auto-merge" "$TMP_DIR/state/bad-effective-used"
+jq '.managedRepositoryRuleset = null' "$POLICY" >"$TMP_DIR/queueless.json"
+run_policy apply "$TMP_DIR/queueless.json" >"$TMP_DIR/queueless.out" 2>&1 || fail "queue-less apply failed: $(cat "$TMP_DIR/queueless.out")"
+[ -f "$TMP_DIR/state/auto-merge" ] && ok "auto-merge enabled without a queue" || fail "queue-less apply left auto-merge disabled"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)